src/main/java/me/xmrvizzy/skyblocker/skyblock/dungeon/DungeonBlaze.java

```diff
@@ -20,92 +22,134 @@
 import java.util.Comparator;
 import java.util.List;
 
+/**
+ * This class provides functionality to render outlines around Blaze entities
+ */
 public class DungeonBlaze {
     private static final Logger LOGGER = LoggerFactory.getLogger(DungeonBlaze.class.getName());
-    private static final float[] WHITE_COLOR_COMPONENTS = { 1.0f, 1.0f, 1.0f };
-    static Entity highestBlaze = null;
-    static Entity lowestBlaze = null;
-    static Entity nextHighestBlaze = null;
-    static Entity nextLowestBlaze = null;
-    static boolean renderHooked = false;
-    
+    private static final float[] WHITE_COLOR_COMPONENTS = {1.0f, 1.0f, 1.0f};
+    private static final QuadColor outlineColorGreen = QuadColor.single(0.0F, 1.0F, 0.0F, 1f);
```

Comment:
```suggestion
    private static final QuadColor outlineColorGreen = QuadColor.single(0.0f, 1.0f, 0.0f, 1.0f);
```
for coherecence with the other values